cmake/modules/TmpCPack.cmake

```diff
@@ -0,0 +1,24 @@
+set(CPACK_PACKAGE_DIRECTORY "${CMAKE_BINARY_DIR}/_packages")
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+set(CPACK_PACKAGING_INSTALL_PREFIX /usr)
+
+set(CPACK_PACKAGE_CONTACT "Ignat Sergeev")
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+
+set(CPACK_DEBIAN_RUNTIME_PACKAGE_NAME "libtmp")
+set(CPACK_DEBIAN_DEVELOPMENT_PACKAGE_NAME "libtmp-dev")
+set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)
+
+set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
+set(CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS ON)
+set(CPACK_DEBIAN_DEVELOPMENT_PACKAGE_DEPENDS
+    "libstdc++-dev, libtmp (= ${CPACK_PACKAGE_VERSION})")
+
+set(CPACK_DEBIAN_RUNTIME_DESCRIPTION
+    "Runtime package"
```

Comment:
Shouldn't be there also a project description?

Reply:
Package description is composed of project description and cpack description, so there shouldn`t